queries/lua/textobjects.scm

```diff
@@ -4,7 +4,11 @@
 
 ; call
 
-(function_call) @call.outer (arguments) @call.inner
+(function_call) @call.outer ((arguments) @call.inner (#match? @call.inner "^[^\\(]"))
```

Comment:
@echasnovski Hi there, I was reading through the source documentation on Tree-sitter queries so that I could hopefully also contribute here/fix some queries as the need arises. Most of the other queries in this PR make a decent amount of sense; from what I gather, `#make-range!` just takes a range of nodes from `@_start` to `@_end` (inclusive) and assigns that to the `@call.inner` capture here. However, what is this line 7 supposed to do? It seems like it would just be a second definition for `@call.inner` for Lua files?

Edit: Didn't mean to start a review, just wanted to leave a short question oops ;-;

Reply:
Basically it can be read as written: capture `function_call` as `@call.outer`; capture `arguments` as `@call.inner` only if it matches `"^[^\\(]"` (i.e. doesn't start with `(`). This query accounts for (in my opinion) the most confusing part of Lua - ability to omit parenthesis in function call in case of a single string or table argument. This is needed because "main" query defines `@call.inner` as something strictly inside parenthesis of a function call. 

But thanks for the question, as it prompted me to make this query a little bit cleaner by narrowing `@call.outer` to not overlap with second query.